session/doc.go

```diff
@@ -14,7 +14,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package node implements a node that can run multiple sessions.
-// Each session runs an instance of state channel network client and
-// is owned by the user of the session.
-package node
+// Package session implements a session to which a user can attach its
+// credentials, open and use state channels.  Each session runs an instance of
```

Comment:
The sentence is unclear. Are open and used state channels attached to the session or are state channels opened and used within a session?

Reply:
State channels are opened, used within a session. Have updated in dfea382fa8caf5a8f96bda0464c5589d96e617ce.